<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件服务器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>文件路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtPath"/>
       </item>
       <item>
        <widget class="QToolButton" name="btnFolder">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../app.qrc">
           <normaloff>:/res/folder.ico</normaloff>:/res/folder.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtIp"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>TCP端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtPort"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>HTTP端口:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtHttpPort"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>启动服务</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>停止服务</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>日志：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="edtLog"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../app.qrc"/>
 </resources>
 <connections/>
</ui>
